chord_metadata_service/chord/workflows/metadata.py

```diff
@@ -19,6 +20,8 @@
 
 WORKFLOW_PHENOPACKETS_JSON = "phenopackets_json"
 WORKFLOW_EXPERIMENTS_JSON = "experiments_json"
+WORKFLOW_EXPERIMENTS_EXTENDED_JSON = "experiments_extended_json"
```

Comment:
I think this should have a different name - maybe `WORKFLOW_EXPERIMENTS_JSON_WITH_FILES = "experiments_json_with_files"`